docs/catalyst_cites.bib

```diff
@@ -1,30 +1,22 @@
-@article{doi:10.1021/acs.est.4c12166,
-author = {Law, Jun Wen and Mignone, Bryan K. and Mallapragada, Dharik S.},
-title = {Role of Technology Flexibility and Grid Coupling on Hydrogen Deployment in Net-Zero Energy Systems},
-journal = {Environmental Science \& Technology},
-volume = {59},
-number = {10},
-pages = {4974-4988},
-year = {2025},
-doi = {10.1021/acs.est.4c12166},
-note ={PMID: 40036667},
-URL = {https://doi.org/10.1021/acs.est.4c12166},
-}
-
-@article{Manocha_2025,
-doi = {10.1088/2753-3751/adafab},
-url = {https://dx.doi.org/10.1088/2753-3751/adafab},
-year = {2025},
-month = {feb},
-publisher = {IOP Publishing},
-volume = {2},
-number = {1},
-pages = {015011},
-author = {Manocha, Aneesha and Mantegna, Gabriel and Patankar, Neha and Jenkins, Jesse D},
-title = {Reducing transmission expansion by co-optimizing sizing of wind, solar, storage and grid connection capacity},
-journal = {Environmental Research: Energy},
-abstract = {Expanding transmission capacity is likely a bottleneck that will restrict variable renewable energy (VRE) deployment required to achieve ambitious emission reduction goals. Interconnection and inter-zonal transmission buildout may be displaced by the optimal sizing of VRE to grid connection capacity and by the co-location of VRE and battery resources behind interconnection. However, neither of these capabilities is commonly captured in macro-energy system models. We develop two new functionalities to explore the substitutability of storage for transmission and the optimal capacity and siting decisions of renewable energy and battery resources through 2030 in the Western Interconnection of the United States. Our findings indicate that modeling optimized interconnection and storage co-location better captures the full value of energy storage and its ability to substitute for transmission. Optimizing interconnection capacity and co-location can reduce total grid connection and shorter-distance transmission capacity expansion on the order of 10% at storage penetration equivalent to 2.5%–10% of peak system demand. The decline in interconnection capacity corresponds with greater ratios of VRE to grid connection capacity (an average of 1.5–1.6 megawatt (MW) PV:1 MW inverter capacity, 1.2–1.3 MW wind:1 MW interconnection). Co-locating storage with VREs also results in a 9%–13% increase in wind capacity, as wind sites tend to require longer and more costly interconnection. Finally, co-located storage exhibits higher value than standalone storage in our model setup (up to ∼43%–45%). Given the coarse representation of transmission networks in our modeling, this outcome likely overstates the real-world importance of storage co-location with VREs. However, it highlights how siting storage in grid-constrained locations can maximize the value of storage and reduce transmission expansion.}
-}
+@misc{Buchsbaum_Kahn-Lang_2025,
```

Comment:
Should probably have a different ID for this entry vs. the next one.  Looks like a cut and paste error.